hl/c++/examples/ptExampleFL.cpp

```diff
@@ -72,7 +72,7 @@ main(void)
     if (err < 0)
         fprintf(stderr, "Error getting packet count.");
 
-    printf("Number of packets in packet table after five appends: %llu\n", count);
+    printf("Number of packets in packet table after five appends: %" PRIdHSIZE "\n", count);
```

Comment:
count is hsize_t and should use the format specifier in H5public.h

Reply:
Should this be PRIuHSIZE instead?

Reply:
Yes. Good catch!